src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -27,7 +27,21 @@
         #input-control="{ isValid }"
         :message-key="`block/${block.uuid}/config/set_contact_property/property_key`">
         <div class="block-contact-property-key">
+          <div v-if="subscriberPropertyFields">
```

Comment:
@safydy-r This needs to be checking that this is empty rather than falsy. We could have the getter default to returning null but that would break the importer I believe and it's better to have a consistently typed getter. 

I've pushed a commit to fix this and bumped voto5 so I can move this to QA. Feel free to tweak my approach though.